pages/login_page.py

```diff
@@ -19,10 +19,16 @@ def login(self, user):
         self.find_element(self.LOGIN_BUTTON).click()
 
     def get_error_message(self):
-        return self.ERROR_MSG
+        return self.find_element(self.ERROR_MSG)
```

Comment:
Honestly its magic) I don't know how it works:
FORGOT_PASSWORD_BUTTON = (By.XPATH, '//*[@id="app"]/div[1]/div/div[1]/div/div[2]/div[''2]/form/div[4]/p')
div[''2] - redundant quotation marks